packages/fxa-auth-server/test/local/ip_profiling.js

```diff
@@ -76,6 +78,7 @@ describe('IP Profiling', function () {
   this.timeout(30000);
 
   beforeEach(() => {
+    Container.set(CapabilityService, {});
```

Comment:
Unrelated test, but `yarn test test/local/` is failing with node FATAL ERROR (hoping to talk with you about that), and this test was failing when run as a smaller chunk without these fixes.

Reply:
Vijay has a fix up for this: https://github.com/mozilla/fxa/pull/13022. More context in that PR.